hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -721,4 +721,24 @@ private Constants() {
    * Default change detection require version: true.
    */
   public static final boolean CHANGE_DETECT_REQUIRE_VERSION_DEFAULT = true;
+
+  public static final String MULTIPART_DOWNLOAD_ENABLED =
+      "fs.s3a.multipartdownload.enabled";
+  public static final boolean DEFAULT_MULTIPART_DOWNLOAD_ENABLED = false;
+
+  public static final String MULTIPART_DOWNLOAD_PART_SIZE =
+      "fs.s3a.multipartdownload.part-size";
+  public static final long DEFAULT_MULTIPART_DOWNLOAD_PART_SIZE = 8_000_000;
+
+  public static final String MULTIPART_DOWNLOAD_CHUNK_SIZE =
+      "fs.s3a.multipartdownload.chunk-size";
+  public static final long DEFAULT_MULTIPART_DOWNLOAD_CHUNK_SIZE = 262_144;
```

Comment:
the javadoc should explain the difference between part size and chunk size, also why a new configuration property is needed instead of reusing fs.s3a.multipart.size